packages/wrangler/src/pages/functions/buildPlugin.ts

```diff
@@ -46,6 +47,9 @@ export function buildPlugin({
 			betaD1Shims: (betaD1Shims || []).map(
 				(binding) => `${D1_BETA_PREFIX}${binding}`
 			),
+			betaConstellationShims: (betaConstellationShims || []).map(
+				(binding) => `${CONSTELLATION_BETA_PREFIX}${binding}`
```

Comment:
Is it in Beta or Alpha? The message pushed in Validation says Alpha.

Reply:
I have, hopefully, removed all "alpha" references.